src/test/java/com/wikia/webdriver/testcases/discussions/DeleteAllPostsByUserTests.java

```diff
@@ -41,8 +41,7 @@
    * @return post that was created
    */
   private PostEntity.Data setUp(String wikiName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "setUp" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)